security/entity_provider.rst

```diff
@@ -169,6 +169,12 @@ forces the class to have the five following methods:
 
 To learn more about each of these, see :class:`Symfony\\Component\\Security\\Core\\User\\UserInterface`.
 
+.. caution::
+
+    Do not actually implement ``eraseCredentials`` when you load your users directly
```

Comment:
Maybe change this to warn that you shouldn't clear data here that are mapped to the underlying database?